### DISCUSSION TITLE
Allow to copy link to live docs

### DISCUSSION THREAD

**Discussion by igorwojda** [GitbookIO/community#342] [Feature Requests]
Scenario 1
- I am logged into the "Editing Dashboard"
- I am editing the docs page
- Add the ability (new menu item) to Edit docs on the GitBook website here:
![image](https://github.com/GitbookIO/community/assets/530122/fd5a4e99-f553-4968-a494-79688f52ec0f)

Currently copied link is linked to the "docs edit"

**Comment by sandstrom** [comment]
We have the same issue! This should be the link to the published page, since it's shared with customers etc. Doesn't make any sense to share an internal link that only signed in users can access.

See screenshot of the link that should be the public/accessible link (our collection has the "Publish as unlisted" state, if that matters).

Also, would be create if this command was in the command panel `ctrl + K`. See "Linear" or "Github" for an example of how to make a great command panel.

![image](https://github.com/GitbookIO/community/assets/122287/5a101beb-8a78-4e1b-b655-e4406ad0fe5e)
